docs/source/examples/shapes_images_positions.rst

```diff
@@ -33,6 +50,42 @@ Shapes, Images and Positions
             self.play(Rotating(dot, about_point=[2, 0, 0]), run_time=1.5)
             self.wait()
 
+Download the resource for the following example `here <https://github.com/ManimCommunity/manim/blob/master/docs/source/_static/manim-logo-sidebar.svg>`_ 
+
+.. manim:: ManimCELogoFromSVG
+
+    class ManimCELogoFromSVG(Scene):
+        def construct(self):
+            v_image = SVGMobject(file_name="manim-logo-sidebar.svg")
+            self.add(v_image)
+
+            # Custom colors can be defined with hex strings
+            logo_blue, logo_green, logo_red = "#525893", "#87C2A5", "#E07A5F"
+
+            # An SVG file with multiple objects can be decomposed into
+            # their subcomponents
+            triangle = v_image.submobjects[0]
+            square = v_image.submobjects[1]
+            circle = v_image.submobjects[2]
+            m = v_image.submobjects[3]
+
+            self.play(
+                ApplyMethod(triangle.set_fill, MANIMCE_LOGO_RED),
```

Comment:
This is causeing the error in the docs build NameError: name 'MANIMCE_LOGO_RED' is not defined
instead it should be logo_red